src/main/java/com/intuit/graphql/orchestrator/datafetcher/FieldResolverDirectiveDataFetcher.java

```diff
@@ -6,20 +6,26 @@
 import graphql.schema.DataFetcher;
 import graphql.schema.DataFetchingEnvironment;
 import java.util.Objects;
+import lombok.Getter;
 
-public class FieldResolverDirectiveDataFetcher implements DataFetcher<Object> {
+@Getter
+public class FieldResolverDirectiveDataFetcher implements ServiceAwareDataFetcher<Object> {
```

Comment:
I'm thinking of standardizing to `... implement `ServiceAwareDataFetcher<CompletableFuture<DataFetcherResult<Object>>>`.

FieldResolverBatrchLoader.java has this signature

```
public CompletionStage<List<DataFetcherResult<Object>>> load(final List<DataFetchingEnvironment> dataFetchingEnvironments)
```